src/test/java/SetTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set의 size() 메소드를 활용해 Set의 크기를 확인")
+    void sizeTest() {
+        int expected = 3;
+
+        assertThat(numbers.size()).isEqualTo(expected);
```

Comment:
컬렉션 사이즈를 검증하는 메소드를 사용해보면 어떨까요?
```suggestion
        assertThat(numbers).hasSize(3);
```